components/x-teaser-timeline/src/TeaserTimeline.jsx

```diff
@@ -0,0 +1,65 @@
+import { h } from '@financial-times/x-engine';
+import { ArticleSaveButton } from '@financial-times/x-article-save-button';
+import { Teaser, presets } from '@financial-times/x-teaser';
+import { getGroupAndIndex, getItemGroups } from './lib/transform';
+import styles from './TeaserTimeline.scss';
+import classNames from 'classnames';
+
+const TeaserTimeline = props => {
+	const {
+		csrfToken = null,
+		customSlotContent,
+		customSlotPosition = 2,
+		items,
+		showSaveButtons = true
+	} = props;
+	const itemGroups = getItemGroups(props);
+
+	if (customSlotContent) {
+		const insertPosition = Math.min(customSlotPosition, items.length);
+		const insert = getGroupAndIndex(itemGroups, insertPosition);
+
+		itemGroups[insert.group].items.splice(insert.index, 0, customSlotContent);
+	}
+
+	return itemGroups.length > 0 && (
+		<div>
+			{itemGroups.map(group => (
+				<section key={group.date} className={classNames(styles.itemGroup)}>
+					<h2 className={classNames(styles.itemGroup__heading)}>{group.title}</h2>
+					<ul className={classNames(styles.itemGroup__items)}>
+						{group.items.map(item => {
+							if (item.id) {
+								return (
+									<li key={item.id} className={styles.item}>
+										<Teaser
+											{...item}
+											{...presets.SmallHeavy}
+											theme="timeline-teaser"
```

Comment:
Is `theme` chosen over `modifiers` so that it always "wins"? If so I think a comment is required, otherwise probably better to use `modifiers` as this will override opinion styling at the very least.

Reply:
I'm pretty sure there isn't a good reason to use `theme`, and `modifiers` seems more appropriate. I'll change it.